_includes/specials/taproot/en/08-signature-adaptors.md

```diff
@@ -0,0 +1,111 @@
+almost the normal way, with the one difference being that they
+essentially generate their nonce in two parts: a true random nonce that
+they will forever keep secret, and their favorite number---which they'll
+initially keep secret but which is safe for other people to discover.
+The donor generates a fully valid signature using both of these values,
+adding them together as if they were a single nonce.
+
+BIP340 signature commitments use the nonce in two forms: a numeric
+representation (called a *scalar*), which normally only the signer
+knows, and as a *point* on the Elliptic Curve (EC), which is published
+to enable verification.
+
+The donor takes the commitment part of their valid signature and
+subtracts out the hidden scalar.  This makes the signature incomplete
+(and thus invalid) but allows the donor to share the (invalid) signature
+commitment, the (valid) point for the complete nonce, and the (valid)
+point for the hidden number.  Together these three pieces of information
+are a *signature adaptor*.
+
+Using a slight variant on the BIP340 signature verification algorithm,
+anyone can verify that signature adaptor would provide a valid signature
+if the hidden scalar was simply added back in to the (currently invalid)
+signature commitment.  This is possible to verify even without knowing
+what that hidden number is.  In short, it is now possible for users to
+trustlessly begin making guesses about the value of hidden scalar,
+secure in the knowledge that a correct guess will allow them to get the
+signature and send the transaction.
+
+Like everyone else who received the donor's signature adaptor, Alice and
+Bob now have a copy of the EC point for the hidden number.  Also like
+everyone else, they don't know the actual scalar.  But, if you recall,
+all the donor did to turn their valid signature into an invalid
+signature is subtract out the hidden number from their signature
+commitment while continuing to have the signature commit to the point
+of the hidden number.  Alice can just as easily create an invalid
+signature by not committing to the scalar she doesn't know but still
+committing to the EC point she does know.  This allows Alice to create a
+signature adaptor for a transaction that pays Bob.  If Bob learns the
+scalar, he can convert that adaptor into a valid signature and send the
+transaction, winning the bet.
+
+But how does Bob learn the winning number?  Does he have wait for
+someone else who guesses it to publish a press release?  Nope.  Recall
+one more time that the signature adaptor the donor published was their
+actual signature minus the scalar.  When the hidden number is discovered
+and somebody sends the 1,000 BTC transaction, they must publish the
+original (valid) signature commitment.  Bob can take that (valid)
+signature commitment and subtract from it the (invalid) signature
+commitment in the original signature adaptor to get the scalar.  He then
```

Comment:
The donor encumbered the signature by deducting the hidden scalar "hs": `sig - hs = invalid_sig`
`invalid_sig - sig = (sig - hs) -sig = -hs`

=> Shouldn't this be "valid signature - invalid signature": `sig - invalid_sig = sig - (sig - hs) = hs`?

Reply:
I think that's what it says?  "take ... valid ... and subtract from it ... invalid"

Reply:
Oh, oops. Sorry, I guess I must have misread that as "take the valid signature commitment and subtract it from the invalid", i.e. switched the order reading of "from" and "it" when reading it.

Reply:
Yeah, the writing here is a bit stilted.  In English, we usually write "subtract b from a" but I've always found that confusing since we write "a - b".  I guess by trying to normalize things, I may have made it more confusing.  I think I'll leave it this way, though.  (Maybe next time I should try phrasing it in RPN: "take the invalid_sig and the valid_sig and perform subtraction"  :-)